xrmocap/core/evaluation/metrics/pcp_metric.py

```diff
@@ -125,21 +125,26 @@ def get_limbs(self, pred_keypoints3d: Keypoints,
         for frame_idx, limb_name in enumerate(limb_name_list):
             if limb_name in selected_limbs_name:
                 ret_limbs.append(conn_list[frame_idx])
+                ret_limb_name.append(limb_name)
+            else:
+                self.logger.error(f'Wrong bone name: {limb_name}')
```

Comment:
`limb_name` not in `selected_limbs_name` is just not selected, it might not be wrong. Shall we use `logger.info` or `logger.warning` here instead?

Reply:
We use `logger.info`.